db/migrate/20150312215548_create_videos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVideos < ActiveRecord::Migration
+  def change
+    create_table :videos do |t|
+      t.string :video_id
+      t.string :youtube_link
```

Comment:
Naming it `youtube_link` is kind of misleading, since we're really only storing the ID part of it. You should probably rename this.